media/static-bundles.json.orig

```diff
@@ -1,1960 +0,0 @@
-{
```

Comment:
I don't see this file in the `main` branch, which is weird. Does this branch need a rebase perhaps?